### ISSUE TITLE
No instances detected on input image while running demo.py

### ISSUE THREAD

**Issue by ghost** [youngwanLEE/centermask2#18]
I run the following command to run a demo -
"python demo.py --input trio.jpg --output results/ --config-file ../configs/centermask/centermask_V_99_eSE_FPN_ms_3x.yaml"
But I get the following-
[03/17 07:05:17 detectron2]: Arguments: Namespace(confidence_threshold=0.5, config_file='../configs/centermask/Base-CenterMask-VoVNet.yaml', input=['trio.jpg'], opts=[], output='results/', video_input=None, webcam=False)
WARNING [03/17 07:05:17 d2.config.compat]: Config '../configs/centermask/Base-CenterMask-VoVNet.yaml' has no VERSION. Assuming it to be compatible with latest v2.
  0% 0/1 [00:00<?, ?it/s][03/17 07:05:22 detectron2]: trio.jpg: detected 0 instances in 0.16s
100% 1/1 [00:00<00:00,  2.61it/s]
And the output is the same as input. There are no segmentation masks painted on the result image.

**Comment by youngwanLEE** [comment]
Did you set a pretrained weight with `MODEL.WEIGHTS path/to/the/centermask-v-99-eSE-ms-3x.pth`?

**Comment by ghost** [comment]
If I give MODEL.WEIGHTS centermask2-V-99-eSE-FPN-ms-3x.pth as an argument to demo.py, I get error "demo.py: error: unrecognized arguments: MODEL.WEIGHTS centermask2-V-99-eSE-FPN-ms-3x.pth"

**Comment by youngwanLEE** [comment]
@karnabhc 
Follow [the instruction](https://github.com/facebookresearch/detectron2/blob/master/GETTING_STARTED.md#inference-demo-with-pre-trained-models) by detectron2 because we use the same demo program.

**Comment by ghost** [comment]
"python demo.py --input trio.jpg --output results/ --config-file ../configs/centermask/Base-CenterMask-VoVNet.yaml --opts MODEL.WEIGHTS centermask2-V-99-eSE-FPN-ms-3x.pth"
Still I get the following-
[03/17 08:00:53 detectron2]: Arguments: Namespace(confidence_threshold=0.5, config_file='../configs/centermask/Base-CenterMask-VoVNet.yaml', input=['trio.jpg'], opts=['MODEL.WEIGHTS', 'centermask2-V-99-eSE-FPN-ms-3x.pth'], output='results/', video_input=None, webcam=False)
WARNING [03/17 08:00:55 d2.config.compat]: Config '../configs/centermask/Base-CenterMask-VoVNet.yaml' has no VERSION. Assuming it to be compatible with latest v2.
  0% 0/1 [00:00<?, ?it/s][03/17 08:01:08 detectron2]: trio.jpg: detected 0 instances in 0.17s
100% 1/1 [00:02<00:00,  2.75s/it]
Can you please help me?

**Comment by youngwanLEE** [comment]
@karnabhc 

You missed the config-file.

`configs/centermask/centermask_V_99_eSE_FPN_ms_3x.yaml`

try below command:
> python demo.py --input trio.jpg --output results/ --config-file configs/centermask/centermask_V_99_eSE_FPN_ms_3x.yaml --opts MODEL.WEIGHTS centermask2-V-99-eSE-FPN-ms-3x.pth

**Comment by ghost** [comment]
Thanks a lot, that solved the issue!
